docs/source/install.rst

```diff
@@ -8,11 +8,12 @@ Installation
 Using setuptools
 ~~~~~~~~~~~~~~~~
 
-To obtain the latest released version of statsmodels using `setuptools <https://pypi.python.org/pypi/setuptools>`__::
+To obtain the latest released version of statsmodels using pip
 
     pip install -U statsmodels
 
-Or follow `this link to our PyPI page <https://pypi.python.org/pypi/statsmodels>`__.
+Or follow `this link to our PyPI page <https://pypi.python.org/pypi/statsmodels>`__, download
+the wheel or source and install.
 
 Statsmodels is also available in `Anaconda <https://www.continuum.io/downloads>`__::
 
```

Comment:
Better to point to conda forge I think: `conda install -c conda-forge statsmodels`.